Dist_pc1727.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_pc1727(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return x^4
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return np.sqrt(0.2857)
+		
```

Comment:
you should use an equation here. something that involves self.x_min and self.x_max.

You should also add the code for running the tests that I sent in an email.